src/Partition.cxx

```diff
@@ -80,7 +80,7 @@ PrefetchSong(InputCacheManager &cache, const char *uri) noexcept
 static void
 PrefetchSong(InputCacheManager &cache, const DetachedSong &song) noexcept
 {
-	PrefetchSong(cache, song.GetURI());
+	PrefetchSong(cache, song.GetRealURI());
```

Comment:
The commit message isn't good - it doesn't explain what needs to be fixed, and why this change fixes the issue.